### PR TITLE
Following release of nuxeo 8.10

### PR DESCRIPTION
**Pull request by dmetzler**

### CODE REVIEW

**Comment by tianon** [comment]
```diff
diff --git a/nuxeo_8.3/Dockerfile b/nuxeo_FT/Dockerfile
similarity index 96%
copy from nuxeo_8.3/Dockerfile
copy to nuxeo_FT/Dockerfile
index 8a91014..22bbc56 100644
--- a/nuxeo_8.3/Dockerfile
+++ b/nuxeo_FT/Dockerfile
@@ -33,8 +33,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ghostscript \
  && rm -rf /var/lib/apt/lists/*
 
-ENV NUXEO_VERSION 8.3
-ENV NUXEO_MD5 6f4a5d5a1df1b024ac2aa27570dd3447
+ENV NUXEO_VERSION 8.10
+ENV NUXEO_MD5 29e67a19bba54099093b51d892926be1
 ENV NUXEO_HOME /opt/nuxeo/server
 
 # Add distribution
diff --git a/nuxeo_7.10/docker-entrypoint.sh b/nuxeo_FT/docker-entrypoint.sh
similarity index 100%
copy from nuxeo_7.10/docker-entrypoint.sh
copy to nuxeo_FT/docker-entrypoint.sh
```

**Comment by tianon** [comment]
Looks like something's not right here :sweat_smile:

Build test of #2440; be88f29 (`nuxeo`):

```console

$ bashbrew build nuxeo:latest
Building bashbrew/cache:cd10db088087f0ef91ad3f322dd2ff6d5b707eeb265453c19a456a4f9bce7f87 (nuxeo:latest)
Tagging nuxeo:latest
Tagging nuxeo:LTS
Tagging nuxeo:LTS-2016
Tagging nuxeo:8
Tagging nuxeo:8.10
Tagging nuxeo:FT

$ test/run.sh nuxeo:latest
testing nuxeo:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]......passed


$ bashbrew build nuxeo:8.3
Using bashbrew/cache:8fa15848a5f0b2b90c374d822b61fbea47ebee799ff413f00813bfc7fe824104 (nuxeo:8.3)
Tagging nuxeo:8.3

$ test/run.sh nuxeo:8.3
testing nuxeo:8.3
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:8.2
Using bashbrew/cache:f10bb2a28e6192c0330511b4d1675e887fe20854f51113cb0a8e7d0c6a90e94a (nuxeo:8.2)
Tagging nuxeo:8.2

$ test/run.sh nuxeo:8.2
testing nuxeo:8.2
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......nuxeo:8.2 stopped unexpectedly!
++ docker logs 4b276a4710ffac536a0d4b8e9a82202f52246442fcbca4797b66a5c15588e266
/docker-entrypoint.sh: ignoring /docker-entrypoint-initnuxeo.d/*

Detected Tomcat server.
Nuxeo home:          /opt/nuxeo/server
Nuxeo configuration: /opt/nuxeo/server/bin/nuxeo.conf
Include template: /opt/nuxeo/server/templates/common-base
Include template: /opt/nuxeo/server/templates/common
Include template: /opt/nuxeo/server/templates/default
Could not run configuration: Nuxeo requires Java 1.8.0_40+ (detected 1.8.0_111). See 'jvmcheck' option to bypass version check.
failed


$ bashbrew build nuxeo:LTS-2015
Using bashbrew/cache:55bfb7d08ec0e20c95e3736c3e4bac03ec071db5875de870af6e79e71bfe3140 (nuxeo:LTS-2015)
Tagging nuxeo:LTS-2015
Tagging nuxeo:7
Tagging nuxeo:7.10

$ test/run.sh nuxeo:LTS-2015
testing nuxeo:LTS-2015
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]......nuxeo:LTS-2015 stopped unexpectedly!
++ docker logs 2a68e26ed985602b0fd7f421912f5805fccff3e8ff1e9e0626f959a3b0d79a35
/docker-entrypoint.sh: ignoring /docker-entrypoint-initnuxeo.d/*

Detected Tomcat server.
Nuxeo home:          /opt/nuxeo/server
Nuxeo configuration: /opt/nuxeo/server/bin/nuxeo.conf
Include template: /opt/nuxeo/server/templates/common-base
Include template: /opt/nuxeo/server/templates/common
Include template: /opt/nuxeo/server/templates/default
Could not run configuration: Nuxeo requires Java 1.8.0_40+ (detected 1.8.0_111). See 'jvmcheck' option to bypass version check.
failed


$ bashbrew build nuxeo:LTS-2014
Using bashbrew/cache:f9b3f7b69792a8f881fcc30d1e38673260e6861fa0a775394b2a925fb5a76033 (nuxeo:LTS-2014)
Tagging nuxeo:LTS-2014
Tagging nuxeo:6
Tagging nuxeo:6.0

$ test/run.sh nuxeo:LTS-2014
testing nuxeo:LTS-2014
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed

```

The following images failed at least one test: nuxeo:8.2 nuxeo:LTS-2015

**Comment by yosifkit** [comment]
Build test of #2440; f1e5e4d (`nuxeo`):

```console

$ bashbrew build nuxeo:latest
Using bashbrew/cache:cd10db088087f0ef91ad3f322dd2ff6d5b707eeb265453c19a456a4f9bce7f87 (nuxeo:latest)
Tagging nuxeo:latest
Tagging nuxeo:LTS
Tagging nuxeo:LTS-2016
Tagging nuxeo:8
Tagging nuxeo:8.10
Tagging nuxeo:FT

$ test/run.sh nuxeo:latest
testing nuxeo:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]......passed


$ bashbrew build nuxeo:8.3
Using bashbrew/cache:8fa15848a5f0b2b90c374d822b61fbea47ebee799ff413f00813bfc7fe824104 (nuxeo:8.3)
Tagging nuxeo:8.3

$ test/run.sh nuxeo:8.3
testing nuxeo:8.3
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:8.2
Using bashbrew/cache:f10bb2a28e6192c0330511b4d1675e887fe20854f51113cb0a8e7d0c6a90e94a (nuxeo:8.2)
Tagging nuxeo:8.2

$ test/run.sh nuxeo:8.2
testing nuxeo:8.2
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:LTS-2015
Using bashbrew/cache:55bfb7d08ec0e20c95e3736c3e4bac03ec071db5875de870af6e79e71bfe3140 (nuxeo:LTS-2015)
Tagging nuxeo:LTS-2015
Tagging nuxeo:7
Tagging nuxeo:7.10

$ test/run.sh nuxeo:LTS-2015
testing nuxeo:LTS-2015
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:LTS-2014
Using bashbrew/cache:f9b3f7b69792a8f881fcc30d1e38673260e6861fa0a775394b2a925fb5a76033 (nuxeo:LTS-2014)
Tagging nuxeo:LTS-2014
Tagging nuxeo:6
Tagging nuxeo:6.0

$ test/run.sh nuxeo:LTS-2014
testing nuxeo:LTS-2014
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed

```

File: test/tests/nuxeo-basics/run.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ NUXEO_TEST_SLEEP=5
 NUXEO_TEST_TRIES=10
 
 cname="nuxeo-container-$RANDOM-$RANDOM"
-cid="$(docker run -d --name "$cname" "$image")"
+cid="$(docker run -d -e LAUNCHER_DEBUG='-Djvmcheck=nofail' --name "$cname" "$image")"
```
Comment (dmetzler) [inline test/tests/nuxeo-basics/run.sh]: I was wondering on if I had to fix the image or not as it's a bug in our code. Meaning if you download the regular ZIP, you have that bug with JVM >= 1.8_u100. I wanted to have the same behavior even if it's a bug.
From a user perspective, I agree that it's better to fix it in the image.

Comment (dmetzler) [inline test/tests/nuxeo-basics/run.sh]: Currently fixint the image by applying https://jira.nuxeo.com/browse/NXP-20189

**Comment by dmetzler** [comment]
changes not finished, i have to point to a JDK for 8.10 so don't merge for now (sorry)

**Comment by tianon** [comment]
No worries -- do what you need to do :+1:

**Comment by dmetzler** [comment]
Done. Should be ok for now

**Comment by tianon** [comment]
LGTM

```diff
diff --git a/nuxeo_7.10/Dockerfile b/nuxeo_7.10/Dockerfile
index 4afc544..92db9aa 100644
--- a/nuxeo_7.10/Dockerfile
+++ b/nuxeo_7.10/Dockerfile
@@ -49,6 +49,7 @@ RUN curl -fsSL "http://cdn.nuxeo.com/nuxeo-${NUXEO_VERSION}/nuxeo-cap-${NUXEO_VE
     && mv /tmp/nuxeo-distribution/$DISTDIR $NUXEO_HOME \
     && sed -i -e "s/^org.nuxeo.distribution.package.*/org.nuxeo.distribution.package=docker/" $NUXEO_HOME/templates/common/config/distribution.properties \
     && rm -rf /tmp/nuxeo-distribution* \
+    && sed -i "s/LAUNCHER_DEBUG /LAUNCHER_DEBUG -Djvmcheck=nofail /" $NUXEO_HOME/bin/nuxeoctl \
     && chmod +x $NUXEO_HOME/bin/*ctl $NUXEO_HOME/bin/*.sh
 
 RUN mkdir /docker-entrypoint-initnuxeo.d
diff --git a/nuxeo_8.2/Dockerfile b/nuxeo_8.2/Dockerfile
index c0ee39d..7e948bb 100644
--- a/nuxeo_8.2/Dockerfile
+++ b/nuxeo_8.2/Dockerfile
@@ -49,6 +49,7 @@ RUN curl -fsSL "http://cdn.nuxeo.com/nuxeo-${NUXEO_VERSION}/nuxeo-cap-${NUXEO_VE
     && mv /tmp/nuxeo-distribution/$DISTDIR $NUXEO_HOME \
     && sed -i -e "s/^org.nuxeo.distribution.package.*/org.nuxeo.distribution.package=docker/" $NUXEO_HOME/templates/common/config/distribution.properties \
     && rm -rf /tmp/nuxeo-distribution* \
+    && sed -i "s/LAUNCHER_DEBUG /LAUNCHER_DEBUG -Djvmcheck=nofail /" $NUXEO_HOME/bin/nuxeoctl \
     && chmod +x $NUXEO_HOME/bin/*ctl $NUXEO_HOME/bin/*.sh
 
 RUN mkdir /docker-entrypoint-initnuxeo.d
diff --git a/nuxeo_8.3/Dockerfile b/nuxeo_FT/Dockerfile
similarity index 95%
copy from nuxeo_8.3/Dockerfile
copy to nuxeo_FT/Dockerfile
index 8a91014..f14e528 100644
--- a/nuxeo_8.3/Dockerfile
+++ b/nuxeo_FT/Dockerfile
@@ -1,5 +1,5 @@
 # vim:set ft=dockerfile:
-FROM       java:8-jre
+FROM       java:8-jdk
 MAINTAINER Nuxeo <packagers@nuxeo.com>
 
 # Create Nuxeo user
@@ -33,8 +33,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ghostscript \
  && rm -rf /var/lib/apt/lists/*
 
-ENV NUXEO_VERSION 8.3
-ENV NUXEO_MD5 6f4a5d5a1df1b024ac2aa27570dd3447
+ENV NUXEO_VERSION 8.10
+ENV NUXEO_MD5 29e67a19bba54099093b51d892926be1
 ENV NUXEO_HOME /opt/nuxeo/server
 
 # Add distribution
diff --git a/nuxeo_7.10/docker-entrypoint.sh b/nuxeo_FT/docker-entrypoint.sh
similarity index 100%
copy from nuxeo_7.10/docker-entrypoint.sh
copy to nuxeo_FT/docker-entrypoint.sh
```

Build test of #2440; 459f7af (`nuxeo`):

```console

$ bashbrew build nuxeo:latest
Building bashbrew/cache:8222542c586dfeced95672e396e3ff9bd5b342cf1b48dea0a72c575ceb4d8324 (nuxeo:latest)
Tagging nuxeo:latest
Tagging nuxeo:LTS
Tagging nuxeo:LTS-2016
Tagging nuxeo:8
Tagging nuxeo:8.10
Tagging nuxeo:FT

$ test/run.sh nuxeo:latest
testing nuxeo:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:8.3
Building bashbrew/cache:1e876f29f863a8c903b715b3a099d5f1fb8c087a34f737d82af3185a7bfdd72b (nuxeo:8.3)
Tagging nuxeo:8.3

$ test/run.sh nuxeo:8.3
testing nuxeo:8.3
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:8.2
Building bashbrew/cache:b1b78a63de47cd78f20f51997e2be1afc473042ea67b9ca68d3b569c02c047b5 (nuxeo:8.2)
Tagging nuxeo:8.2

$ test/run.sh nuxeo:8.2
testing nuxeo:8.2
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:LTS-2015
Building bashbrew/cache:eb9775feaed5ddb25d50d88d7da970ef73f9e05846760891cc7367725daa5d38 (nuxeo:LTS-2015)
Tagging nuxeo:LTS-2015
Tagging nuxeo:7
Tagging nuxeo:7.10

$ test/run.sh nuxeo:LTS-2015
testing nuxeo:LTS-2015
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed


$ bashbrew build nuxeo:LTS-2014
Using bashbrew/cache:f9b3f7b69792a8f881fcc30d1e38673260e6861fa0a775394b2a925fb5a76033 (nuxeo:LTS-2014)
Tagging nuxeo:LTS-2014
Tagging nuxeo:6
Tagging nuxeo:6.0

$ test/run.sh nuxeo:LTS-2014
testing nuxeo:LTS-2014
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6].......passed

```

**Comment by tianon** [comment]
Getting the following on the official build server while trying to build this:
```
Step 8 : ENV NUXEO_VERSION 8.3
 ---> Using cache
 ---> c0c85fe778e0
Step 9 : ENV NUXEO_MD5 6f4a5d5a1df1b024ac2aa27570dd3447
 ---> Using cache
 ---> 9798c0db4a19
Step 10 : ENV NUXEO_HOME /opt/nuxeo/server
 ---> Using cache
 ---> eeeb76e26658
Step 11 : RUN curl -fsSL "http://cdn.nuxeo.com/nuxeo-${NUXEO_VERSION}/nuxeo-server-${NUXEO_VERSION}-tomcat.zip" -o /tmp/nuxeo-distribution-tomcat.zip     && echo "$NUXEO_MD5 /tmp/nuxeo-distribution-tomcat.zip" | md5sum -c -     && mkdir -p /tmp/nuxeo-distribution $(dirname $NUXEO_HOME)     && unzip -q -d /tmp/nuxeo-distribution /tmp/nuxeo-distribution-tomcat.zip     && DISTDIR=$(/bin/ls /tmp/nuxeo-distribution | head -n 1)     && mv /tmp/nuxeo-distribution/$DISTDIR $NUXEO_HOME     && sed -i -e "s/^org.nuxeo.distribution.package.*/org.nuxeo.distribution.package=docker/" $NUXEO_HOME/templates/common/config/distribution.properties     && rm -rf /tmp/nuxeo-distribution*     && chmod +x $NUXEO_HOME/bin/*ctl $NUXEO_HOME/bin/*.sh
 ---> Running in b21715a5c537
md5sum: WARNING: 1 computed checksum did NOT match
/tmp/nuxeo-distribution-tomcat.zip: FAILED
Removing intermediate container b21715a5c537
The command '/bin/sh -c curl -fsSL "http://cdn.nuxeo.com/nuxeo-${NUXEO_VERSION}/nuxeo-server-${NUXEO_VERSION}-tomcat.zip" -o /tmp/nuxeo-distribution-tomcat.zip     && echo "$NUXEO_MD5 /tmp/nuxeo-distribution-tomcat.zip" | md5sum -c -     && mkdir -p /tmp/nuxeo-distribution $(dirname $NUXEO_HOME)     && unzip -q -d /tmp/nuxeo-distribution /tmp/nuxeo-distribution-tomcat.zip     && DISTDIR=$(/bin/ls /tmp/nuxeo-distribution | head -n 1)     && mv /tmp/nuxeo-distribution/$DISTDIR $NUXEO_HOME     && sed -i -e "s/^org.nuxeo.distribution.package.*/org.nuxeo.distribution.package=docker/" $NUXEO_HOME/templates/common/config/distribution.properties     && rm -rf /tmp/nuxeo-distribution*     && chmod +x $NUXEO_HOME/bin/*ctl $NUXEO_HOME/bin/*.sh' returned a non-zero code: 1
```

Any ideas for what could be causing this?  Was the tarball updated again already?

**Comment by dmetzler** [comment]
Fingerprint has changed, on 12/08 because of https://jira.nuxeo.com/browse/NXP-21267 (damn jvmCheck bug !!!) 
I'm sending another PR

**Comment by dmetzler** [comment]
https://github.com/docker-library/official-images/pull/2450

**Pull request merged**